### PR TITLE
Enable saving prediction output

### PR DESCRIPTION
**Pull request by kashyapchhatbar**
- Added argument and save by default
- Include all tags in output file for multimer preset

### CODE REVIEW

File: inference.py [hunk 1]
```diff
@@ -500,6 +506,9 @@ def inference_monomer_model(args):
                         type=str,
                         default='full_dbs',
                         choices=('reduced_dbs', 'full_dbs'))
+    parser.add_argument('--save_prediction_result',
```
Comment (Gy-Lu) [inline inference.py:509]: This argument seems not to be used.

File: inference.py [hunk 2]
```diff
@@ -312,7 +318,7 @@ def inference_multimer_model(args):
 
     # Save the relaxed PDB.
     relaxed_output_path = os.path.join(args.output_dir,
-                                        f'{tag}_{args.model_name}_relaxed.pdb')
+        f'{output_prefix}_{args.model_name}__relaxed.pdb')
```
Comment (Gy-Lu) [inline inference.py:321]: Why there are 2 '_' before 'relaxed'?

File: inference.py [hunk 3]
```diff
@@ -282,6 +281,13 @@ def inference_multimer_model(args):
 
     out = result_q.get()
 
+
+    # Save the prediction result .pkl
+    prediction_result_path = os.path.join(args.output_dir,
+        f'{output_prefix}_{args.model_name}_.pkl')
+    with open(prediction_result_path, 'wb') as f:
+        pickle.dump(out, f, protocol=4)
```
Comment (Gy-Lu) [inline inference.py:289]: Why not just keep the protocol default?

File: inference.py (hunk 1 above)
Comment (kashyapchhatbar) [reply inference.py:509]: Should be fixed now..

File: inference.py (hunk 2 above)
Comment (kashyapchhatbar) [reply inference.py]: Thanks for spotting the typo, should be fixed now.

File: inference.py (hunk 3 above)
Comment (kashyapchhatbar) [reply inference.py]: I agree, I have now modified it to default.

**Comment by kashyapchhatbar** [comment]
I have modified the pull request, thanks for reviewing it. Hopefully, it is a useful addition.

**Review by Gy-Lu — APPROVED** [review]

**Pull request merged**